packages/resource/README.md

```diff
@@ -65,6 +66,24 @@ const pages = createAggregated(currentPage, []);
 
 Objects and Arrays are re-created on each operation, but the values will be left untouched, so `<For>` should work fine.
 
+### createDeepSignal
+
+This is an optimized implementation of what is required by the [resource storage option](https://www.solidjs.com/docs/latest/api#createresource:~:text=Resources%20can%20be%20set%20with%20custom%20defined%20storage):
+
+```ts
+// this adds fine-grained reactivity to the contents of data():
+const [data, { refetch }] = createResource(fetcher, { storage: createDeepSignal });
+```
+
+> ⚠ if your resource is a deep signal, you can no longer rely on reactive changes to the base signal. If you want to combine this with [`createAggregated`](#createaggregated), you will need to wrap the resource to either call [`deepTrack`](https://solid-primitives.netlify.app/package/deep#deeptrack) or read one of the reactive parts that you know for certain will change every time:
```

Comment:
```suggestion
> **Warning** if your resource is a deep signal, you can no longer rely on reactive changes to the base signal. If you want to combine this with [`createAggregated`](#createaggregated), you will need to wrap the resource to either call [`deepTrack`](https://solid-primitives.netlify.app/package/deep#deeptrack) or read one of the reactive parts that you know for certain will change every time:
```
Github mas a special styling for these: https://github.com/orgs/community/discussions/16925
Our website will too